modules/jt-comparing-jenkins-tekton-concepts.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-comparing-jenkins-tekton-concepts_{context}"]
+= Comparing Jenkins and Tekton concepts
+
+toc::[]
+
+
+This section summarizes the basic terms used in Jenkins and Tekton, and compares the equivalent terms.
+
+== Jenkins terminology
+Jenkins offers declarative and scripted pipelines that are extensible using shared libraries and plugins. Some basic terms in Jenkins are as follows:
+
+* *Pipeline*: Automates the entire process of building, testing, and deploying applications, using the link:https://groovy-lang.org/[Groovy] syntax.
+* *Node*: A machine capable of orchestrating and/or executing a scripted pipeline.
+* *Stage*: A conceptually distinct subset of tasks performed in a pipeline. This block is often used by plugins or user interfaces to display status or progress of tasks.
```

Comment:
Optional rephrase (don't feel obligated to make the change). I think passive voice reads fine here. However, if you want to avoid passive voice whenever you can, I think this rephrase reads just as well: 
`Plugins or user interfaces often use this block to display status or progress of tasks.`